collections/hub/golden_path/feature_tests/transfer_negative_scenarios/payee_invalid_fulfillment.json

```diff
@@ -165,200 +169,94 @@
                 "exec": [
                   "expect(response.statusText).to.equal('Accepted')"
                 ]
-              },
-              {
-                "id": 3,
-                "description": "Callback Validate Protected Headers",
```

Comment:
I do think we need to add a story to add tests specifically for validating JWS, protected headers, etc.

Reply:
I believe other tests cover jws and protected headers. If you think these need it as well, I can make the story.